queries/liquidityPools/useCurveSusdPoolQuery.ts

```diff
@@ -133,12 +131,17 @@ const useCurveSusdPoolQuery = (options?: QueryConfig<CurveData>) => {
 				curveRewards,
 				curveStaked,
 				curveAllowance,
-			].map((data) => Number(toBigNumber(data.toString()).div(1e18)));
+			].map((data) => wei(data));
 
-			const curveRate =
-				(((inflationRate * relativeWeight * 31536000) / workingSupply) * 0.4) / curveSusdTokenPrice;
-			const rewardsAPR = curveRate * curvePrice * 1e18;
-			const swapAPR = swapData?.data?.apy?.day?.susd ?? 0;
+			const curveRate = inflationRate
+				.mul(relativeWeight)
+				.mul(31536000)
```

Comment:
maybe we should be using a ONE_YEAR constant for that one, since it's being used in more than one file.

Reply:
just following up on this @KillerByte